public/styles.css

```diff
@@ -203,18 +203,16 @@
 }
 .showDefault-questions {
   position: relative;
-  max-height: 250px;
+  max-height: 500px;
   margin: 2%;
   border: solid black thin;
 
 }
 .showDefault-answers {
   min-height: 50px;
   margin: 2%;
-
-
 }
-.showDefault-questions div:nth-child(5n) {
+.showDefault-questions div:nth-child(4n) {
```

Comment:
Clever selectors here. This probably isn't applicable to this specific line of code, but I've been finding the > CSS selector useful for selecting children of particular elements.